app/Http/Middleware/Role.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace App\Http\Middleware;
+
+use Closure;
+use Illuminate\Http\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Illuminate\Support\Facades\Auth; 
+
+class Role
```

Comment:
What's the difference between Password and Role middlewares?

Reply:
removed Password middleware